includes/plugins/class-jetpack.php

```diff
@@ -307,12 +310,28 @@ public static function remove_google_analytics_from_available( $modules ) {
 	 *
 	 * @param string $old_val_or_opt_name If adding will be opt name. If updating will be old value.
 	 * @param string $new_val Value correlated to update/add.
-	 * @return void 
+	 * @return void
 	 */
 	public static function set_default_modules( string $old_val_or_opt_name, string $new_val ) {
 		if ( $new_val === '1' ) {
 			update_option( 'jetpack_active_modules', static::$default_active_modules );
 		}
 	}
+
+	/**
+	 * Restrict the age of related content shown by Jetpack Related Posts.
+	 *
+	 * @param array $date_range Array of start and end dates.
+	 * @return array Filtered array of start/end dates.
+	 */
+	public static function restrict_age_of_related_posts( $date_range ) {
+		$related_posts_max_age = get_option( ( new \Newspack\Wizards\Newspack\Recirculation_Section() )->related_posts_option );
```

Comment:
I think we can change the `related_posts_option` to a class constant, it's how we usually treat these, and it avoids instantiating a class that, IMO, should be a singleton.

We're also in the Newspack namespace here, you can call `Wizards\Newspack\Recirculation_Section`

Reply:
Thanks, that's indeed more elegant. Updated in f2a326c